### ISSUE TITLE
After each VSCode update, updated remote code-server doesn't remove previous version

### ISSUE THREAD

**Issue by pawelzwronek** [microsoft/vscode#228141]
Type: <b>Bug</b>

### OS/Web Information

- Local OS: Win10
- Remote OS: Debian GNU/Linux 12 (bookworm), Linux 6.6.31+rpt-rpi-v8
- Remote Architecture: Raspberry Pi 5, arm64
- `code-server --version`: 1.93.0

### Steps to Reproduce

1. Connect to raspberry over ssh(public/private key). Wait for code-server get installed (once)
2. When vscode gets new update, it update code-server on the raspberry, and old version of code-server stays on the raspberry. Now I have 5 older version and 1 the latest, each 180MB

### Expected

Old version should be auto-removed 

### Actual

Old versions of code-server stays on the server

VS Code version: Code 1.93.0 (4849ca9bdf9666755eb463db297b69e5385090e3, 2024-09-04T13:02:38.431Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 3700X 8-Core Processor              (16 x 3600)|
|GPU Status|2d_canvas: unavailable_software<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software<br>webgpu: unavailable_software<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.92GB (16.76GB free)|
|Process Argv|--folder-uri file:///d%3A/git/notepad-plus-plus|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (78)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
quasar-snippets|abd|1.0.0
Bookmarks|ale|13.5.0
vscode-color|ans|0.4.5
icons-carbon|ant|0.2.6
asciidoctor-vscode|asc|3.3.1
vscode-custom-css|be5|7.2.2
arm|dan|1.7.4
vscode-eslint|dba|3.0.10
gitlens|eam|15.4.0
EditorConfig|Edi|0.16.4
el-vsc-product-icon-theme|ElA|0.1.8
vscode-great-icons|emm|2.1.106
prettier-vscode|esb|11.0.0
vscode-todo-plus|fab|4.19.1
vscode-pyvmmonitor|fab|0.0.3
copilot|Git|1.229.0
copilot-chat|Git|0.20.0
triggertaskonsave|Gru|0.2.17
toggler|hid|0.7.0
vue-snippets|hol|1.0.4
vscode-htmlhint|HTM|1.0.5
search-editor-apply-changes|jak|0.1.1
vscode-icon-theme|jtl|1.6.6
Lisp|mat|0.1.12
code-point|med|1.11.0
language-gettext|mro|0.5.0
csdevkit|ms-|1.10.16
csharp|ms-|2.45.20
vscode-dotnet-runtime|ms-|2.1.5
debugpy|ms-|2024.10.0
isort|ms-|2023.10.1
pylint|ms-|2023.10.1
python|ms-|2024.14.0
vscode-pylance|ms-|2024.9.1
jupyter|ms-|2024.8.1
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.19
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
remote-ssh|ms-|0.114.2
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.88.2
cmake-tools|ms-|1.19.51
cpptools|ms-|1.21.6
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.10.0
js-debug-nightly|ms-|2024.9.517
live-server|ms-|0.4.14
powershell|ms-|2024.2.2
remote-explorer|ms-|0.4.3
vscode-js-profile-flame|ms-|1.0.9
vshaxe|nad|2.32.1
folder-archiver|pda|0.0.5
material-icon-theme|PKi|5.10.0
java|red|1.34.0
vscode-xml|red|0.27.1
LiveServer|rit|5.7.9
docxreader|Sha|1.1.3
svg-preview|Sim|2.8.3
code-spell-checker|str|3.0.1
code-spell-checker-polish|str|2.1.2
sass-indented|syl|1.8.31
vscode-autohotkey2-lsp|thq|2.5.2
graphviz-interactive-preview|tin|0.3.5
pdf|tom|1.2.2
gnu-mapfiles|tro|1.1.0
cmake|twx|0.0.17
errorlens|use|3.20.0
vscodeintellicode|Vis|1.3.1
vscode-gradle|vsc|3.16.4
vscode-java-debug|vsc|0.58.0
vscode-java-dependency|vsc|0.24.0
vscode-java-pack|vsc|0.29.0
vscode-maven|vsc|0.44.0
volar|Vue|2.1.6
vscode-autohotkey-debug|zer|1.11.0
linkerscript|Zix|1.0.4

(2 theme extensions excluded)

</details>

**Comment by GongT** [comment]
It work as intent. Maybe you need this: https://www.freedesktop.org/software/systemd/man/latest/systemd-tmpfiles.html

**Comment by pawelzwronek** [comment]
> It work as intent. Maybe you need this: https://www.freedesktop.org/software/systemd/man/latest/systemd-tmpfiles.html

You are right. I looked at the source code, and the installation script keeps the last 5 installations. Also, I should have submitted this issue to https://github.com/microsoft/vscode-remote-release
